x-pack/plugin/eql/src/test/java/org/elasticsearch/xpack/eql/planner/QueryFolderFailTests.java

```diff
@@ -62,53 +62,61 @@ public void testBetweenWrongTypeParams() {
                 error("process where between(process_name, \"s\", \"e\", false, 2)"));
     }
 
+    public void testMatchWithText() {
```

Comment:
No test for regexes?

Reply:
added more in 44af6a6dd84d5ff45ac0ee8b26462d88236b21b9